awx/ui/src/screens/NotificationTemplate/shared/TypeInputsSubForm.js

```diff
@@ -408,10 +409,21 @@ function TwilioFields() {
         label={t`Destination SMS number(s)`}
         name="notification_configuration.to_numbers"
         type="textarea"
-        validate={required(null)}
+        validate={combine([required(null), twilioPhoneNumber()])}
         isRequired
-        tooltip={t`Enter one phone number per line to specify where to
-          route SMS messages.`}
+        tooltip={
+          <>
+            {t`Enter one phone number per line to specify where to
+          route SMS messages. Phone numbers should be formatted +11231231234. For more information see `}
+            <a
+              target="_blank"
+              href="https://www.twilio.com/docs/verify/migrating-1x-2x#e164-phone-number-formatting-for-sms-and-voice"
```

Comment:
If the third party docs link changes, we will have a broken link in our application - I would suggest not adding a link to a certain URL.